.github/workflows/main.yml

```
@@ -57,15 +55,25 @@ jobs:
         run: |
           pipenv check
         continue-on-error: true
-      - uses: hadolint/hadolint-action@v2.1.0
-        with:
-          dockerfile: docker/Dockerfile
       - name: Run tests
         run: |
           cd exodus
           python manage.py test
         env:
           DJANGO_SETTINGS_MODULE: exodus.settings.dev
+
+  hadolint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: hadolint/hadolint-action@v2.1.0
+        with:
+          dockerfile: docker/Dockerfile
+
+  docker_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Why separate lint and build about Docker ?
Use one workflow is more efficient for this (You build on Docker only if lint is ok)
 ```suggestion
   docker_build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: hadolint/hadolint-action@v2.1.0
        with:
          dockerfile: docker/Dockerfile
```

---

Using one workflow is definitely not more efficient, as the new way would allow to do both in parallel (so, more efficiently).

If we want to only build the Docker image if the lint is ok (which can make sense, I actually hesitated to do that), I would still offer to do it in 2 separate workflows (with the `need` keyword) so that we can clearly identify which task fails and how much time each task is taking.

@Jean-BaptisteC Let me know if you think we should have this approach (only build the Docker image if the lint passes).

---

Ok to add `need`